src/getting-started.md

```diff
@@ -9,75 +9,270 @@ SRI stack is flexible. It allows you to run a few different configurations. The
+
+```log
+[2023-03-20 12:32:06] DEBUG: job_id='1' extranonce2=0000000000000000000000000000 ntime=6417ba1a
+[2023-03-20 12:32:06] Stratum requested work restart
+[2023-03-20 12:32:12] DEBUG: hash <= target
+Hash:   00000001658bd854fa337fc9a8eb76981750146e18a8daae3c1b8468bd1c921d
+Target: 0000000dfff20000000000000000000000000000000000000000000000000000
+[2023-03-20 12:32:12] > {"method": "mining.submit", "params": ["", "1", "0000000000000000000000000000", "6417ba1a", "ad3fbde1"], "id":4}
+[2023-03-20 12:32:12] < {"id":4,"error":null,"result":true}
+[2023-03-20 12:32:12] accepted: 1/1 (100.00%), 51235 khash/s (yay!!!)
+```
+
+Eventually, the Translation Proxy log output will show sucessful share, which means you've run the configuration successfully!
+
+```log
+2023-03-21T17:54:24.665389Z  INFO translator_sv2::downstream_sv1::downstream: PROXY SERVER - ACCEPTING FROM DOWNSTREAM: 127.0.0.1:36298
+2023-03-21T17:54:24.665855Z  INFO translator_sv2::downstream_sv1::downstream: Down: Subscribing
+2023-03-21T17:54:24.666843Z  INFO translator_sv2::downstream_sv1::downstream: Down: Authorizing
+BITCOIN TARGET: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+2023-03-21T17:54:28.721866Z  INFO translator_sv2::proxy::bridge: SHARE MEETS TARGET
+2023-03-21T17:54:28.722207Z  INFO translator_sv2::upstream_sv2::upstream: Up: Sending: Sv2Frame { header: Header { extension_type: 32768, msg_type: 27, msg_length: U24(41) }, payload: Some(Mining(SubmitSharesExtended(SubmitSharesExtended { channel_id: 1, sequence_number: 0, job_id: 1, nonce: 2784102488, ntime: 1679417936, version: 536870912, extranonce: Owned([0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]) }))), serialized: None }
+2023-03-21T17:54:29.584107Z  INFO translator_sv2::upstream_sv2::upstream: Is future job: true
+
+2023-03-21T17:54:29.584210Z  INFO translator_sv2::upstream_sv2::upstream: Up: New Extended Mining Job
+2023-03-21T17:54:29.584363Z  INFO translator_sv2::upstream_sv2::upstream: Up: Set New Prev Hash
+[roles/translator/src/proxy/bridge.rs:584] job.job_id = 2
+[roles/translator/src/proxy/bridge.rs:584] sv2_set_new_prev_hash.job_id = 2
+BITCOIN TARGET: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+2023-03-21T17:54:33.437332Z  INFO translator_sv2::proxy::bridge: SHARE MEETS TARGET
+```
+
+
+## Config D: SV1 firmware > Translation Proxy JN (Job Negotiator) > SV2 Pool
+
+This configuration allows mining devices running SV1 firmware to connect to an SV2 Pool through a Translation Proxy (tProxy). In this case the proxy is designed also to implement the **Job Negotiator (JN)** features: it's able to negotiate the block templates to mine on with the **Pool-side JN**. In this configuration, **transaction selection** is done by the miners locally (through the tProxy JN) and then negotiated with the pool.
```

Comment:
There's not really any negotiation happening. In current implementation, the pool cannot decline what miners propose, so I re-worded it for clarity.

```suggestion
This configuration allows mining devices running SV1 firmware to connect to an SV2 Pool through a Translation Proxy (tProxy). In this case the tProxy is designed also to implement the **Job Negotiator (JN)** sub-protocol: allowing miners to select transaction locally and send them the **Pool-side JN**. 
```

Reply:
Totally agree with you. 
Thanks!